### ISSUE TITLE
App crashes instantly on ios 12.5.5, ipad mini 2

### ISSUE THREAD

**Issue by kai-to-prwto-hakeraki** [nextcloud/ios#1897]
### Steps to reproduce

1. Have the last version of Nextcloud downloaded on an ipad mini 2 running ios 12.5.5
2. Try to open it
3. Observe

### Expected behaviour

It should open.

### Actual behaviour

It shows a blue screen with the white logo on the centre and then it closes.

### Logs

```
If applicable, you can post the iOS app or server logs (removing any sensitive information).
```

### Environment data

**iOS version:**  iOS 12.5.5

**Nextcloud iOS app version:** 4.2.1 

**Server operating system:**

**Web server:** Apache, nginx

**Database:**

**PHP version:**

**Nextcloud version:** see Nextcloud admin page

**Comment by adb336** [comment]
### Steps to reproduce
1. install Nextcloud App from Apple Appstore
2. Configure with Credentials
3. Start App again
4. App logo appears and vanishes after some seconds
5. App still active in background
6. no interaction possible
7. App cannot be used any longer from other apps as store location

### Expected behaviour
- App should start showing directories and files

### Actual behaviour
- after logo comes up App disappears again after some seconds
- to be noticed: clients on Android and Windows 10 are not having issues with the same server on the same net
- Nextcloud WebUI is useable

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

Cannot reproduce sind this environment cannot be reached with my iPad.

### Environment data
iOS version: 12.5.5

Device model: first generation iPad

Stock or customized system:

Nextcloud app version: iOS version 4.2.0.5

Nextcloud server version: 22.2.3 in the official snap für Ubuntu LTS 20.04

Reverse proxy: nope

### Logs
#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by czjjwx** [comment]
I have the same issue. CANNOT OPEN THE NEXTCLOUD APP.
ipad air 12.5.5  model MF016LL/A
NEXTCLOUD VER. 4.2.1

**Comment by marinofaggiana** [comment]
Fixed V 4.2.2
